fre/pp/split_netcdf_script.py

```diff
@@ -242,11 +257,19 @@ def set_coord_encoding(dset, vcoords):
 def set_var_encoding(dset, varnames):
   '''
   Gets the encoding settings needed for xarray to write out the variables
-  as expected
-  mostly addressed to time_bnds, because xarray can drop the units attribute:
+  as expected::
+  
+  mostly addressed to time_bnds, because xarray can drop the units attribute
     https://github.com/pydata/xarray/issues/8368
```

Comment:
This line needs to have 2 less spaces (align with the text above it) so Sphinx stays happy